lib/media/media_source_engine.js

```diff
@@ -223,13 +223,14 @@ shaka.media.MediaSourceEngine.prototype.destroy = function() {
  *   'video': 'video/webm; codecs="vp9"', 'text': 'text/vtt' }.
  *   All types given must be supported.
  * @param {boolean} useRelativeCueTimestamps
+ * @param {Object} overrideCueRegion
```

Comment:
This should be a strong type, such as VTTRegion.  You may need to define VTTRegion in externs/.